<launch>
  <arg name="config_filepath" default="$(find-pkg-share doughnut_calib)/warthog.config.yaml" />

  <!-- node pkg="doughnut_calib" name="keyboard_ramp_control" type="doughnut_keyboard.py">
  </node -->

  <node pkg="doughnut_calib" name="doughnut_calib_node" exec="doughnut_calib_node" output="screen">
    <param from="$(var config_filepath)" />
    <remap from="joy_in" to="joy"/>
    <remap from="cmd_vel_out" to="doughnut_cmd_vel"/>
    <remap from="left_wheel_in" to="/left_drive/velocity"/>
    <remap from="right_wheel_in" to="/right_drive/velocity"/>
  </node>

<!--  <executable cmd="ros2 bag record -a -o /media/robot/ssd-2nd/rosbag/doughnut" />-->

</launch>
